Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+
+// @LambdaSchool
+function HeaderContent() {
+  return <p>Let's learn React by building simple interfaces with components.  Don't try to overthink it, just keep it simple and have fun.  Once you feel comfortable using components you are well on your way to mastering React!</p>;
+}
+//
+// timestamp?
```

Comment:
you can use the date object for a timestamp. See here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date